ProblemsSolution.Tests/ProblemsSolutionTest.cs

```diff
@@ -0,0 +1,32 @@
+using NUnit.Framework;
+
+namespace ProblemsSolution.Tests
+{
+    [TestFixture]
+    public class Tests
+    {
+
+        [TestCase (new int[] {1,2,3,4,5}, ExpectedResult=6, Description="Check for array with even numbers.")]
+        [TestCase (new int[] {3, 7, 0, 1}, ExpectedResult=0, Description="Check for array with even numbers and a zero.")]
```

Comment:
The description is incorrect.

Reply:
Fixed 3a3cecea212765927f6183cdb4f8273d4c578ee9.